Processing_data/Classes/Coordinate.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Processing_data.Classes
+{
+    /// <summary>
+    /// Represents a geographical coordinate
+    /// </summary>
+    public struct Coordinate
+    {
+        public double Latitude { get; set; }
+        public double Longitude { get; set; }
+
+        public Coordinate(double latitude, double longitude)
```

Comment:
no need for the constructor. The structure may be initialized as `new Coordinate { Longitude = x, Latitude = y };`

Reply:
updated